### PR TITLE
MHR: Location Types UXA

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#13119

*Description of changes:*
* Misc location type UXA updates, descriptions listed in story ticket.
* Other misc bug fix which was deleting `locationType` from payload at point of MHR submission under ticket:
 /bcgov/entity#14874

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by ChasBoucher — APPROVED** [review]
Looks Good

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Comment by dimak1** [comment]
I was just reviewing it, but then it got merged 😄

**Comment by cameron-eyds** [comment]
@dimak1 Oh, sorry! Don't hesitate to complete your review, can make any adjustments in a follow up commit 👍

File: ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue [hunk 1]
```diff
@@ -247,6 +290,12 @@ export default defineComponent({
           default:
             return '(Not Entered)'
         }
+      }),
+      hasManualEntries: computed((): boolean => {
+        const location = getMhrRegistrationLocation.value
+        return !!location.lot || !!location.parcel || !!location.block || !!location.districtLot || !!location.partOf ||
```
Comment (dimak1) [inline ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue:296]: I was thinking if we could optimize here `Object.values(location).every(val => !!val)`

**Review by dimak1 — COMMENTED** [review]
Just one comment

**Comment by dimak1** [comment]
@cameron-eyds all good, PR looks good, with just one comment.

File: ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue:296]: Totally could if every property stored on `location` in this instance was to be considered as part of this computed condition, however it's only a select few unfortunately.

Comment (dimak1) [reply ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue:296]: I see, yea I didn't check if those were all of the props.. makes sense, thanks!

**Pull request merged**